kubernetes-with-otelcol-tracing/README.md

```diff
@@ -0,0 +1,27 @@
+# Buildbarn cache with traces and metrics collection
+
+This deployment is Datadog-centric, but it ought to be fairly straightforward to swap out the Datadog-specific bits for any other upstream that supports Opentelemetry.
+
+## Deployment Attributes
+
+1. sharded local storage, with each pod having 250GB for CAS, 250GB for AC, and 5GB for persistence
+2. traces being sent to Datadog via the opentelemetry collector in daemon mode. The trace sampler is set to "always" because the preferred approach is to send all traces to Datadog, and then sample down based on your cost needs on the datadog side. This is because analytics data comes from all ingested spans, not just the retained ones.
+3. scrape prometheus metrics to Datadog into the metrics namespace "buildbarn". This includes the metrics for the Opentelemetry Collector, so you can have some visibility into ingestion rate, errors, etc.
+
+## Further reading
+
+- Datadog prometheus scraping configuration: https://docs.datadoghq.com/agent/kubernetes/prometheus/
+- Datadog span retention: https://docs.datadoghq.com/tracing/trace_retention_and_ingestion
+
+## Usage Notes
```

Comment:
Could you document real quick how to set this up? See for instance [how the existing Kubernetes deployment](https://github.com/buildbarn/bb-deployments/tree/master/kubernetes#buildbarn-deployment-for-kubernetes) is documented.